cloud_function/main.py

```diff
@@ -100,7 +100,7 @@ def submit_aou_workload(event, context):
     input_data = json.loads(manifest_file_content)
     notification = input_data['notifications'][0]
     input_values = notification.values()
-    input_files = set([f for f in input_values if str(f).startswith(f"gs://{bucket}")])
+    input_files = set([f for f in input_values if str(f).startswith(f"gs://{bucket.name}")])
```

Comment:
Based on my local testing, even this is not enough, see:
```python
>>> b = client.bucket("gs://broad-exomes-dev1")
>>> str(b)
'<Bucket: gs://broad-exomes-dev1>'
>>> b.name
'gs://broad-exomes-dev1'
>>>
```

We probably need something like:
```suggestion
    input_files = set([f for f in input_values if str(f).startswith(bucket.name)])
```

Reply:
Actually never mind, the `name` indeed depends on the `event['bucket']` within this context and it's the bucket name without `gs://`. @samanehsan correct me if I'm wrong here 😄

Reply:
Yes, that's right!